src/software/Localization/main_cameraLocalizer.cpp

```diff
@@ -436,7 +436,15 @@ int main(int argc, char** argv)
     {
       POPART_CERR("Unable to localize frame " << frameCounter);
 #if HAVE_ALEMBIC
-      exporter.jumpKeyframe();
+      if(frameCounter == 0)
+      {
+        // No previous pose, use the identity matrix
+        exporter.addCameraKeyframe(geometry::Pose3(), &queryIntrinsics, currentImgName, frameCounter, frameCounter);
+      }
+      else
+      {
+        exporter.jumpKeyframe();
```

Comment:
is there any way in the alembic file to know how many frames there are so far?
This won't solve the problem at API level, if someone calls jumpKeyframe() without having frame the problem remains. I wonder if we can solve it directly inside jumpKeyFrame(), eg. checking how many frame we have so far or something like that.
@caymard @fabiencastan